app/views/common_parameters/_parent_parameter.html.erb

```diff
@@ -7,7 +7,7 @@
           <%= content_tag(:input, '', :class => "form-control", :disabled => true,  :value => _('Global'), :type=> :text ) %>
         </td>
         <td class="col-md-2">
-          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :placeholder => parameter.name) %>
+          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :value => parameter.name) %>
```

Comment:
ok, found it.
current workaround: put a space, it saves as an empty value.

I say - let's put this in, and in the parameters ui PR I will implement the same UI as in puppetclass parameters for this part as well. two birds, one stone.